PaddleCV/image_classification/utils/utility.py

```diff
@@ -88,10 +88,8 @@ def parse_args():
     add_arg('data_dir',                 str,    "./data/ILSVRC2012/",   "The ImageNet dataset root directory.")
     add_arg('pretrained_model',         str,    None,                   "Whether to load pretrained model.")
     add_arg('checkpoint',               str,    None,                   "Whether to resume checkpoint.")
-    add_arg('save_params',              str,    "./output",                   "Whether to save params.")
-
     add_arg('print_step',               int,    10,                     "The steps interval to print logs")
```

Comment:
print_step -> log_step?

Reply:
I follow this [rule](https://github.com/xyzhou-puck/models_new/tree/master/docs)

Reply:
ok